README.md

```diff
@@ -61,13 +77,21 @@ helm install orchestrator orchestrator --set orchestrator.devmode=true \
     --set rhdhOperator.github.token=$GITHUB_TOKEN
 ```
 
-To enable the K8s and Tekton (OpenShift Pipelines) plugins in Backstage, install by:
+To enable the K8s, Tekton (OpenShift Pipelines) and ArgoCD (OpenShift GitOps) plugins in Backstage, install by:
```

Comment:
Do we want to keep adding new entries to this option? :)
Maybe a table with all possible flags and their default value and if they are mandatory may be better (like we do for all the possible values in a chart)
This may be for another PR/ticket